docs/contributing/building_from_source.md

```diff
@@ -25,6 +25,14 @@ Extra steps for Windows users:
 
 ### Prerequisites
 
+You will need to [install Rust](https://www.rust-lang.org/tools/install). Check
+that you have the required tools:
```

Comment:
Can you add that Deno requires the latest stable Rust release and does not support nightly builds?

Reply:
done :)